sormas-backend/src/main/java/de/symeda/sormas/backend/customizableenum/CustomizableEnumFacadeEjb.java

```diff
@@ -143,24 +138,19 @@
 			fillDiseaseCache(type, enumClass, innerDisease);
 		}
 
-		List<T> enumValues = new ArrayList<>();
-		enumValuesByLanguage.get(enumClass).get(language).forEach((value, caption) -> {
-			try {
-				T enumValue = enumClass.getDeclaredConstructor().newInstance();
-				enumValue.setValue(value);
-				enumValue.setCaption(caption);
-				enumValue.setProperties(enumProperties.get(type).get(value));
-				enumValues.add(enumValue);
-			} catch (InstantiationException | IllegalAccessException | NoSuchMethodException | InvocationTargetException e) {
-				throw new RuntimeException(e);
-			}
-		});
+		Stream<String> diseaseValuesStream;
+		if (innerDisease.isPresent()) {
+			// combine specific and unspecific values
+			diseaseValuesStream = Stream.concat(
```

Comment:
Frankly, I do not completely understand what how this is working, but can't we get two enties for the same value now?

Reply:
No, enum values are unique per data type, so it's not possible that the same value is in the steam twice.